src/modules/generateRandomNumber.js

```diff
@@ -7,7 +7,17 @@
  * @return {number} A random 4-digit number
  */
 function generateRandomNumber() {
-  /* Write your code here */
+  const numArr = [Math.floor(Math.random() * 9) + 1];
+
+  while (numArr.length < 4) {
+    const newNum = Math.floor(Math.random() * 9);
```

Comment:
The range for generating `newNum` should be from 0 to 9 inclusive, but `Math.floor(Math.random() * 9)` generates numbers from 0 to 8. To include 9, you should use `Math.floor(Math.random() * 10)`.